hudi-common/src/main/java/org/apache/hudi/common/util/RocksDBDAO.java

```diff
@@ -121,7 +121,7 @@ protected void log(InfoLogLevel infoLogLevel, String logMsg) {
         managedDescriptorMap.put(familyNameFromDescriptor, descriptor);
       }
     } catch (RocksDBException | IOException re) {
-      LOG.error("Got exception opening rocks db instance ", re);
+      LOG.error("Got exception opening Rocks DB instance ", re);
```

Comment:
I would not call this a typo :) .. but sure.. casing makes sense. merging..

Reply:
thanks!